flocker/node/_config.py

```diff
@@ -1311,4 +1346,5 @@ def marshal_configuration(state):
         "version": 1,
         "applications": result,
         "used_ports": sorted(state.used_ports),
+        # XXX "manifestations": ...
```

Comment:
And also need to output additional `datasets` key in global config combined by `flocker-deploy`.